### ISSUE TITLE
Review the accessibility report from LAA and Deque

### ISSUE THREAD

**Issue by karenmajewicz** [geobtaa/geoportal#509]
The BTAA Geoportal was recently evaluated for accessibility. This evaluation was funded by the Library Accessibility Alliance (LAA), a cooperative group of over 100 academic libraries devoted to promoting accessibility of library electronic resources. The evaluation was conducted by a third party accessibility consultant, Deque. The assessment revealed issues with assistive technology compatibility, resulting in some users missing information required to understand content, states and operate functions. 

[The full report is available on the Library Accessibility Alliance’s website.](https://libraryaccessibility.org/evaluation/btaa-geoportal-platform).

**Comment by karenmajewicz** [comment]
During the June GeoBlacklight sprint, we will go through the report and create issues in BTAA, GeoBlacklight, and Blacklight
